### PR TITLE
initial-commit

### PR DESCRIPTION
**Pull request by theVivek-kumar**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+.heading-GE{
+  color: rgb(29, 28, 28);
+  font-size: 3rem;
+}
+.location-heading{
+  color: rgb(255, 255, 255)!important;
+  font-weight: 600;
+  font-size: 1.2rem;
+  z-index: 3;
+   /* position: absolute;
```
Comment (Ananyamadhu08) [inline src/App.css]: you can delete the commented code.

**Review by Ananyamadhu08 — APPROVED** [review]
You can delete the commented code other than that everything looks good, you can go ahead and merge.

File: src/Components/home.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+function Home() {
+    return (
+      <div className='main-container'>
+        <Clock/>
+        <h1 className='heading-GE'>Good Evening, {}.</h1>
+
+        {/* <div style={{display:display}}>
+        <p className="para">What is your main focus for today?</p>
```
Comment (JatuFaique) [inline src/Components/home.js]: You can remove these commented lines of not needed.

**Review by JatuFaique — APPROVED** [review]
The code looks clean and implementation is clear.
Looks good to merge.

File: src/Components/Clock/Clock.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+
+ import { React, useState} from 'react'
```
Comment (Sen-442b) [inline src/Components/Clock/Clock.jsx:2]: You don't need to import `React` here

File: src/Components/Weather.jsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  const apiKey = "f56f24967aaf51182d1d4df628297c6d"
+  const [inputCity, setInputCity] = useState("")
+  const [data, setData] = useState({})
+  const [ display , setDisplay ] = useState("inline")
+
+  const mycity = localStorage.getItem('myCity')
+
+  const getWetherDetails = (cityName) => {
+    if (!cityName) return
+    const apiURL = "https://api.openweathermap.org/data/2.5/weather?q=" + mycity + "&appid=" + apiKey
```
Comment (Sen-442b) [inline src/Components/Weather.jsx:15]: you can use template literals here for better code readability

File: src/Components/home.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+import Weather from './Weather'
+
+
+function Home() {
+    return (
+      <div className='main-container'>
+        <Clock/>
+        <h1 className='heading-GE'>Good Evening, {}.</h1>
+
+        {/* <div style={{display:display}}>
```
Comment (Sen-442b) [inline src/Components/home.js]: remove redundant comments  if possible

File: src/Components/quot.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react'
+
+function Quot() {
```
Comment (Sen-442b) [inline src/Components/quot.js:3]: `Quote` is misspelled

**Review by Sen-442b — APPROVED** [review]
Everything else looks good to me

**Pull request merged**